packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -381,11 +381,21 @@ export class GCFBootstrapper {
   }
 
   // TODO: How do we still get access to this installation token?
-  async getAuthenticatedOctokit(installationId: number): Promise<Octokit> {
+  async getAuthenticatedOctokit(
+    installationId: number,
+    wrapOptions?: WrapOptions
```

Comment:
do we have access to `wrapOptions` when we call `getAuthenticatedOctokit` wondering if we could pass it in?